Extensions/AnchorBehavior/tests/anchorruntimebehavior.spec.js

```diff
@@ -0,0 +1,210 @@
+
+      expect(object.getX()).to.equal(500);
+      expect(object.getY()).to.equal(500);
+      expect(object.getWidth()).to.equal(1010);
+    });
+
+    it('anchors the right edge of object (proportional)', function () {
+      var object = createObject();
+      runtimeGame.setGameResolutionSize(1000, 1000);
+      object.setPosition(500, 500);
+
+      object.getBehavior('Anchor').setAnchor('left', 3);
+
+      runtimeScene.renderAndStep(1000 / 60);
+
+      runtimeGame.setGameResolutionSize(2000, 2000);
+      runtimeScene.renderAndStep(1000 / 60);
+
+      expect(object.getX()).to.equal(1000);
+      expect(object.getY()).to.equal(500);
+      expect(object.getWidth()).to.equal(10);
+    });
+  });
+
+  describe('(anchor vertical edge)', function () {
+    ['top', 'bottom'].forEach((objectEdge) => {
+      it(`anchors the ${objectEdge} edge of object to window top (fixed)`, function () {
+        var object = createObject();
+        runtimeGame.setGameResolutionSize(1000, 1000);
+        object.setPosition(500, 500);
+
+        object.getBehavior('Anchor').setAnchor('top', 1);
+
+        runtimeScene.renderAndStep(1000 / 60);
+
+        runtimeGame.setGameResolutionSize(2000, 2000);
+        runtimeScene.renderAndStep(1000 / 60);
+
+        expect(object.getX()).to.equal(500);
+        expect(object.getY()).to.equal(500);
+        expect(object.getWidth()).to.equal(10);
+      });
+    });
+    ['top', 'bottom'].forEach((objectEdge) => {
+      it(`anchors the ${objectEdge} edge of object to window bottom (fixed)`, function () {
+        var object = createObject();
+        runtimeGame.setGameResolutionSize(1000, 1000);
+        object.setPosition(500, 500);
+
+        object.getBehavior('Anchor').setAnchor('bottom', 2);
+
+        runtimeScene.renderAndStep(1000 / 60);
+
+        runtimeGame.setGameResolutionSize(2000, 2000);
+        runtimeScene.renderAndStep(1000 / 60);
+
+        expect(object.getX()).to.equal(500);
+        expect(object.getY()).to.equal(1500);
+        expect(object.getWidth()).to.equal(10);
+      });
+    });
+
+    it('anchors the top and bottom edge of object (fixed)', function () {
+      var object = createObject();
+      runtimeGame.setGameResolutionSize(1000, 1000);
+      object.setPosition(500, 500);
+
+      object.getBehavior('Anchor').setAnchor('top', 1);
+      object.getBehavior('Anchor').setAnchor('bottom', 2);
+
+      runtimeScene.renderAndStep(1000 / 60);
+
+      runtimeGame.setGameResolutionSize(2000, 2000);
+      runtimeScene.renderAndStep(1000 / 60);
+
+      expect(object.getX()).to.equal(500);
+      expect(object.getY()).to.equal(500);
+      expect(object.getHeight()).to.equal(1010);
+    });
+
+    it('anchors the bottom edge of object (proportional)', function () {
+      var object = createObject();
+      runtimeGame.setGameResolutionSize(1000, 1000);
+      object.setPosition(500, 500);
+
+      object.getBehavior('Anchor').setAnchor('top', 3);
```

Comment:
Same question between the name of the test and the actual call to setAnchor('top')?